code/modules/clothing/under/skirt_dress.dm

```diff
@@ -2,8 +2,8 @@
 	fitted = FEMALE_UNIFORM_TOP
 	can_adjust = FALSE
 	body_parts_covered = CHEST|GROIN
-	icon = 'icons/obj/clothing/under/dress.dmi'
-	worn_icon = 'icons/mob/clothing/under/dress.dmi'
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/dress.dmi' // SKYRAT EDIT
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/dress.dmi'
```

Comment:
you need skyrat edit on both of these so we know both are changed. Yes, we should be capable of inferring that they both are. We are not capable.